Client Startup Crash

### What happened?

Game crashes after pressing play button

### Mod Version

Copperative_1.19.2--0.2.0

### Minecraft Version

1.19

### Relevant log output

```Shell
[17:33:25] [modloading-worker-0/ERROR]: Failed to create mod instance. ModID: copperative, class galena.copperative.Copperative
java.lang.reflect.InvocationTargetException: null
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:68) ~[javafmllanguage-1.19.2-43.2.21.jar%23697!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:121) ~[fmlcore-1.19.2-43.2.21.jar%23696!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
Caused by: java.lang.ExceptionInInitializerError
	at galena.copperative.Copperative.<init>(Copperative.java:46) ~[Copperative_1.19.2--0.2.0.jar%23484!/:0.2.0]
	... 14 more
Caused by: net.minecraft.ResourceLocationException: Non [a-z0-9/._-] character in path of location: copperative:oxıdızed_copper_bricks
	at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:37) ~[client-1.19.2-20220805.130853-srg.jar%23695!/:?]
	at net.minecraft.resources.ResourceLocation.<init>(ResourceLocation.java:46) ~[client-1.19.2-20220805.130853-srg.jar%23695!/:?]
	at net.minecraftforge.registries.DeferredRegister.register(DeferredRegister.java:183) ~[forge-1.19.2-43.2.21-universal.jar%23700!/:?]
	at galena.copperative.index.CBlocks.register(CBlocks.java:136) ~[Copperative_1.19.2--0.2.0.jar%23484!/:0.2.0]
	at galena.copperative.index.CBlocks.registerWeatheringSet(CBlocks.java:164) ~[Copperative_1.19.2--0.2.0.jar%23484!/:0.2.0]
	at galena.copperative.index.CBlocks.registerWeatheringSet(CBlocks.java:170) ~[Copperative_1.19.2--0.2.0.jar%23484!/:0.2.0]
	at galena.copperative.index.CBlocks.<clinit>(CBlocks.java:49) ~[Copperative_1.19.2--0.2.0.jar%23484!/:0.2.0]
	at galena.copperative.Copperative.<init>(Copperative.java:46) ~[Copperative_1.19.2--0.2.0.jar%23484!/:0.2.0]
	... 14 more
[17:33:25] [modloading-worker-0/INFO]: Loading Module Color Runes
[17:33:25] [modloading-worker-0/INFO]: Loading Module Seed Pouch
[17:33:25] [modloading-worker-0/INFO]: Loading Module Bundle Recipe
[17:33:25] [modloading-worker-0/INFO]: Loading Module Ambient Discs
[17:33:25] [modloading-worker-0/INFO]: Loading Module Endermosh Music Disc
[17:33:25] [modloading-worker-0/INFO]: Loading Module Beacon Redirection
[17:33:25] [modloading-worker-0/INFO]: Loading Module Abacus
[17:33:25] [modloading-worker-0/INFO]: Loading Module Bottled Cloud
[17:33:25] [modloading-worker-0/INFO]: Loading Module Ancient Tomes
[17:33:25] [modloading-worker-0/INFO]: Loading Module Pathfinder Maps
[17:33:25] [modloading-worker-0/INFO]: Loading Module Compasses Work Everywhere
[17:33:25] [modloading-worker-0/INFO]: Loading Module Zombie Villagers On Normal
[17:33:25] [modloading-worker-0/INFO]: Loading Module Reacharound Placing
[17:33:25] [modloading-worker-0/INFO]: Loading Module Lock Rotation
[17:33:25] [modloading-worker-0/INFO]: Loading Module Slimes To Magma Cubes
[17:33:25] [modloading-worker-0/INFO]: Loading Module Pat The Dogs
[17:33:25] [modloading-worker-0/INFO]: Loading Module Villagers Follow Emeralds
[17:33:25] [modloading-worker-0/INFO]: Loading Module Automatic Recipe Unlock
[17:33:25] [modloading-worker-0/INFO]: Loading Module Snow Golem Player Heads
[17:33:25] [modloading-worker-0/INFO]: Loading Module Poison Potato Usage
[17:33:25] [modloading-worker-0/INFO]: Loading Module More Note Block Sounds
[17:33:25] [modloading-worker-0/INFO]: Loading Module Sponges Better
[17:33:25] [modloading-worker-0/INFO]: Loading Module Armed Armor Stands
[17:33:25] [modloading-worker-0/INFO]: Loading Module Glass Shard
[17:33:25] [modloading-worker-0/INFO]: Loading Module Vexes Die With Their Masters
[17:33:25] [modloading-worker-0/INFO]: Loading Module Utility Recipes
[17:33:25] [modloading-worker-0/INFO]: Loading Module Sponge On Water Placement
[17:33:25] [modloading-worker-0/INFO]: Loading Module Simple Harvest
[17:33:25] [modloading-worker-0/INFO]: Loading Module Gold Tools Have Fortune
[17:33:25] [modloading-worker-0/INFO]: Loading Module Enhanced Ladders
[17:33:25] [modloading-worker-0/INFO]: Loading Module Renewable Spore Blossoms
[17:33:25] [modloading-worker-0/INFO]: Loading Module Shulker Packing
[17:33:25] [modloading-worker-0/INFO]: Loading Module Better Elytra Rocket
[17:33:25] [modloading-worker-0/INFO]: Loading Module Slabs To Blocks
[17:33:25] [modloading-worker-0/INFO]: Loading Module Safer Creatures
[17:33:25] [modloading-worker-0/INFO]: Loading Module Emotes
[17:33:25] [modloading-worker-0/INFO]: Loading Module Horses Swim
[17:33:25] [modloading-worker-0/INFO]: Loading Module Coral On Cactus
[17:33:25] [modloading-worker-0/INFO]: Loading Module Pig Litters
[17:33:25] [modloading-worker-0/INFO]: Loading Module Grab Chickens
[17:33:25] [modloading-worker-0/INFO]: Loading Module No Durability On Cosmetics
[17:33:25] [modloading-worker-0/INFO]: Loading Module More Banner Layers
[17:33:25] [modloading-worker-0/INFO]: Loading Module Hoe Harvesting
[17:33:25] [modloading-worker-0/INFO]: Loading Module Dragon Scales
[17:33:25] [modloading-worker-0/INFO]: Loading Module Replace Scaffolding
[17:33:25] [modloading-worker-0/INFO]: Loading Module Campfires Boost Elytra
[17:33:25] [modloading-worker-0/INFO]: Loading Module Sign Editing
[17:33:25] [modloading-worker-0/INFO]: Loading Module Double Door Opening
[17:33:25] [modloading-worker-0/INFO]: Loading Module Dyeable Item Frames
[17:33:25] [modloading-worker-0/INFO]: Loading Module Diamond Repair
[17:33:25] [modloading-worker-0/INFO]: Loading Module Monster Box
[17:33:25] [modloading-worker-0/INFO]: Loading Module Spiral Spires
[17:33:25] [modloading-worker-0/INFO]: Loading Module Blossom Trees
[17:33:25] [modloading-worker-0/INFO]: Loading Module Fairy Rings
[17:33:25] [modloading-worker-0/INFO]: Loading Module New Stone Types
[17:33:25] [modloading-worker-0/INFO]: Loading Module Azalea Wood
[17:33:25] [modloading-worker-0/INFO]: Loading Module Ancient Wood
[17:33:25] [modloading-worker-0/INFO]: Loading Module No More Lava Pockets
[17:33:25] [modloading-worker-0/INFO]: Loading Module Glimmering Weald
[17:33:25] [modloading-worker-0/INFO]: Loading Module Chorus Vegetation
[17:33:25] [modloading-worker-0/INFO]: Loading Module Corundum
[17:33:25] [modloading-worker-0/INFO]: Loading Module Nether Obsidian Spikes
[17:33:25] [modloading-worker-0/INFO]: Loading Module Big Stone Clusters
[17:33:25] [modloading-worker-0/INFO]: Loading Module Permafrost
[17:33:25] [modloading-worker-0/INFO]: Loading Module Foxhound
[17:33:25] [modloading-worker-0/INFO]: Loading Module Forgotten
[17:33:25] [modloading-worker-0/INFO]: Loading Module Toretoise
[17:33:25] [modloading-worker-0/INFO]: Loading Module Stonelings
[17:33:25] [modloading-worker-0/INFO]: Loading Module Shiba
[17:33:25] [modloading-worker-0/INFO]: Loading Module Wraith
[17:33:25] [modloading-worker-0/INFO]: Loading Module Crabs
[17:33:25] [modloading-worker-0/INFO]: Loading Module Variant Animal Textures
[17:33:25] [modloading-worker-0/INFO]: Loading Module Elytra Indicator
[17:33:25] [modloading-worker-0/INFO]: Loading Module Camera
[17:33:25] [modloading-worker-0/INFO]: Loading Module Microcrafting Helper
[17:33:25] [modloading-worker-0/INFO]: Loading Module Usage Ticker
[17:33:25] [modloading-worker-0/INFO]: Loading Module Back Button Keybind
[17:33:25] [modloading-worker-0/INFO]: Loading Module Chest Searching
[17:33:25] [modloading-worker-0/INFO]: Loading Module Auto Walk Keybind
[17:33:25] [modloading-worker-0/INFO]: Loading Module Soul Candles
[17:33:25] [modloading-worker-0/INFO]: Loading Module Long Range Pick Block
[17:33:25] [modloading-worker-0/INFO]: Loading Module Greener Grass
[17:33:25] [modloading-worker-0/INFO]: Loading Module Improved Tooltips
[17:33:25] [modloading-worker-0/INFO]: Loading Module Wool Shuts Up Minecarts
[17:33:25] [modloading-worker-0/INFO]: Loading Module Uses For Curses
[17:33:25] [modloading-worker-0/INFO]: Loading Module Adjustable Chat
[17:33:25] [modloading-worker-0/INFO]: Loading Module Spawner Replacer
[17:33:25] [modloading-worker-0/INFO]: Loading Module Narrator Readout
[17:33:25] [modloading-worker-0/INFO]: Loading Module Game Nerfs
[17:33:25] [modloading-worker-0/INFO]: Loading Module Enchantments Begone
[17:33:25] [modloading-worker-0/INFO]: Loading Module Enchantment Predicates
[17:33:25] [modloading-worker-0/INFO]: Loading Module Overlay Shader
[17:33:25] [modloading-worker-0/INFO]: Loading Module Crate
[17:33:25] [modloading-worker-0/INFO]: Loading Module Magnets
[17:33:25] [modloading-worker-0/INFO]: Loading Module Matrix Enchanting
[17:33:25] [modloading-worker-0/INFO]: Loading Module Pipes
[17:33:25] [modloading-worker-0/INFO]: Loading Module Totem Of Holding
[17:33:25] [modloading-worker-0/INFO]: Loading Module Backpack
[17:33:25] [modloading-worker-0/INFO]: Loading Module Tiny Potato
[17:33:25] [modloading-worker-0/INFO]: Dispatching Module Step CONSTRUCT_CLIENT
[17:33:26] [Render thread/FATAL]: Failed to complete lifecycle event CONSTRUCT, 1 errors found
[17:33:26] [Render thread/WARN]: Removed resource pack mod_resources from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack Everycomp Generated Pack from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack Immersive Weathering Generated Pack from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack Snowyspirit Generated Pack from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack Supplementaries Generated Pack from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack V Slab Compat Generated Pack from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/Copper Color Fix 3.0.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/Delightful-Plaques-v1.1.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/Detail-Brush-1.5-1.19.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/EclecticTrove-noconfig-1.18.1-1.1.0.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/EnderEyes_1.19_v2.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/Mandalorian Armor from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/Pixelated_v1.3.1_EnhancedVisuals(nox_cgt).zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/TEXTURES.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/VanillaTweaks_r197025.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/animation from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/eatinganimations+ 1.10.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/enhanced-spyglass-v2-0.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/noobikcat-s-better-end-portal-frame.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/pixelaccurate from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/softfluidsmodcompat-1.18.2-0.1.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/vex-allay.zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/WARN]: Removed resource pack resources/§bEnchanting-Reimagined§8-[v1].zip from options because it doesn't seem to exist anymore
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.sound.SoundEngineLoadEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterColorHandlersEvent$Block to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterColorHandlersEvent$Item to a broken mod state
[17:33:26] [Render thread/INFO]: Hardware information:
[17:33:26] [Render thread/INFO]: CPU: 4x Intel(R) Core(TM) i5-7300HQ CPU @ 2.50GHz
[17:33:26] [Render thread/INFO]: GPU: NVIDIA GeForce GTX 1050/PCIe/SSE2 (Supports OpenGL 3.2.0 NVIDIA 536.23)
[17:33:26] [Render thread/INFO]: OS: Windows 10 (10.0)
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RenderLevelStageEvent$RegisterStageEvent to a broken mod state
[17:33:26] [Render thread/INFO]: Replacing search trees with 'JEI' provider
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterParticleProvidersEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterClientReloadListenersEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$RegisterLayerDefinitions to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$RegisterRenderers to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterTextureAtlasSpriteLoadersEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterClientTooltipComponentFactoriesEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterEntitySpectatorShadersEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterKeyMappingsEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterRecipeBookCategoriesEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterGuiOverlaysEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterDimensionSpecialEffectsEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterNamedRenderTypesEvent to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterColorHandlersEvent$ColorResolvers to a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterItemDecorationsEvent to a broken mod state
[17:33:26] [Render thread/INFO]: Invalidating pack caches
[17:33:26] [Render thread/INFO]: Reloading ResourceManager: Default, quark-emote-pack, Create Re-Created 1.3.zip
[17:33:26] [Render thread/ERROR]: Cowardly refusing generate assets for a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing generate assets for a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing generate assets for a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing generate assets for a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing generate assets for a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing generate assets for a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing generate assets for a broken mod state
[17:33:26] [Render thread/ERROR]: Cowardly refusing generate assets for a broken mod state
[17:33:27] [Worker-ResourceReload-1/ERROR]: Cowardly refusing to send event net.minecraftforge.client.event.ModelEvent$RegisterGeometryLoaders to a broken mod state
[17:33:27] [Render thread/FATAL]: Preparing crash report with UUID a8f3f2a8-093a-4ebb-9b1d-02ac471afd20
[17:33:27] [Render thread/FATAL]: Preparing crash report with UUID ed4f09e7-ff57-4479-91ef-5a30b90f383e
```

--- a/src/main/java/galena/copperative/config/CommonConfig.java
+++ b/src/main/java/galena/copperative/config/CommonConfig.java
@@ -15,6 +15,7 @@ import net.minecraftforge.registries.ForgeRegistries;
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Locale;
 import java.util.Map;
 import java.util.function.BooleanSupplier;
 import java.util.function.Predicate;
@@ -129,7 +130,7 @@ public class CommonConfig {
             var property = builder.define("enabled", true);
             var targets = ImmutableMap.<OverrideTarget, BooleanSupplier>builder();
             for (OverrideTarget target : OverrideTarget.values()) {
-                var targetProperty = builder.define(target.name().toLowerCase(), true);
+                var targetProperty = builder.define(target.name().toLowerCase(Locale.ROOT), true);
                 targets.put(target, targetProperty::get);
             }
 

--- a/src/main/java/galena/copperative/config/OverwriteEnabledCondition.java
+++ b/src/main/java/galena/copperative/config/OverwriteEnabledCondition.java
@@ -11,6 +11,7 @@ import net.minecraftforge.common.crafting.conditions.IConditionSerializer;
 import net.minecraftforge.registries.ForgeRegistries;
 
 import javax.annotation.Nullable;
+import java.util.Locale;
 import java.util.Objects;
 
 public class OverwriteEnabledCondition implements ICondition {
@@ -47,7 +48,7 @@ public class OverwriteEnabledCondition implements ICondition {
         @Override
         public void write(JsonObject json, OverwriteEnabledCondition value) {
             json.addProperty("block", value.key.toString());
-            if (value.target != null) json.addProperty("target", value.target.name().toLowerCase());
+            if (value.target != null) json.addProperty("target", value.target.name().toLowerCase(Locale.ROOT));
         }
 
         @Override

--- a/src/main/java/galena/copperative/data/CLang.java
+++ b/src/main/java/galena/copperative/data/CLang.java
@@ -9,6 +9,7 @@ import net.minecraft.world.level.block.WeatheringCopper;
 import net.minecraftforge.registries.RegistryObject;
 
 import java.util.List;
+import java.util.Locale;
 
 public class CLang extends CLangProvider {
 
@@ -20,7 +21,7 @@ public class CLang extends CLangProvider {
         blocks.forEach(block -> {
             if(block.get() instanceof WeatheringCopper copper && copper.getAge() != WeatheringCopper.WeatherState.UNAFFECTED) {
                 var name = copper.getAge().name();
-                var prefix = name.substring(0, 1).toUpperCase() + name.substring(1).toLowerCase();
+                var prefix = name.substring(0, 1).toUpperCase() + name.substring(1).toLowerCase(Locale.ROOT);
                 addBlock(block, prefix + " " + base);
             } else {
                 addBlock(block, base);

--- a/src/main/java/galena/copperative/data/provider/CItemModelProvider.java
+++ b/src/main/java/galena/copperative/data/provider/CItemModelProvider.java
@@ -15,6 +15,7 @@ import net.minecraftforge.registries.RegistryObject;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Locale;
 import java.util.function.Supplier;
 
 public abstract class CItemModelProvider extends ItemModelProvider {
@@ -151,7 +152,7 @@ public abstract class CItemModelProvider extends ItemModelProvider {
     public String weatherPrefix(Block block) {
         if (block instanceof WeatheringCopper it) {
             var age = it.getAge();
-            if (age != WeatheringCopper.WeatherState.UNAFFECTED) return age.name().toLowerCase() + "_";
+            if (age != WeatheringCopper.WeatherState.UNAFFECTED) return age.name().toLowerCase(Locale.ROOT) + "_";
         }
         return "";
     }

--- a/src/main/java/galena/copperative/index/CBlocks.java
+++ b/src/main/java/galena/copperative/index/CBlocks.java
@@ -149,7 +149,7 @@ public class CBlocks {
 
     public static <B extends Block> CopperSet<B> registerConvertedSet(String name, Supplier<B> targetSupplier, Function<WeatherState, B> function, ResourceKey<CreativeModeTab> tab) {
         var weathered = Stream.of(WeatherState.EXPOSED, WeatherState.WEATHERED, WeatherState.OXIDIZED).<Supplier<B>>map(weatherState -> {
-            String prefix = weatherState.name().toLowerCase() + "_";
+            String prefix = weatherState.name().toLowerCase(Locale.ROOT) + "_";
             return register(prefix + name, () -> function.apply(weatherState), tab);
         }).toList();
         return new CopperSet<>(targetSupplier, weathered);
@@ -164,7 +164,7 @@ public class CBlocks {
         WeatherState[] wStates = WeatherState.values();
         ArrayList<RegistryObject<B>> blocks = new ArrayList<>(4);
         for (final WeatherState weatherState : wStates) {
-            String prefix = weatherState.equals(WeatherState.UNAFFECTED) ? "" : weatherState.name().toLowerCase() + "_";
+            String prefix = weatherState.equals(WeatherState.UNAFFECTED) ? "" : weatherState.name().toLowerCase(Locale.ROOT) + "_";
             Supplier<? extends B> supplier = () -> function.apply(weatherState);
             blocks.add(register(name.apply(prefix), supplier, tab));
         }

